src/Compilers/CSharp/Portable/Compilation/MemberSemanticModel.cs

```diff
@@ -1676,18 +1676,6 @@ internal protected virtual CSharpSyntaxNode GetBindableSyntaxNode(CSharpSyntaxNo
                     case SyntaxKind.AnonymousObjectMemberDeclarator:
                         return GetBindableSyntaxNode(parent);
 
-                    case SyntaxKind.DeclarationExpression:
-                    case SyntaxKind.TupleExpression:
```

Comment:
>case SyntaxKind.TupleExpression: [](start = 20, length = 32)

Are we certain that we can always bind these (both) expressions without knowing context?